.travis.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         - pip install poetry
         - poetry install -v
       script:
-        - pytest
+        - pytest --cov=pyartifactory --cov-branch --cov-report=xml

```

Comment:
Just as a remark, Codacy does not support branch coverage, so this is not needed (but it might support it one day, and it doesn't break with this option, so you can leave it in...)